### DISCUSSION TITLE
چرا محدودیت تعداد آی پی را توی پنل نمیذارید؟

### DISCUSSION THREAD

**Discussion by hiddify-com** [hiddify/Hiddify-Manager#238] [Q&A سوالات رایج] [answered]
من نمیخوام لینکی که به کاربر ها میدم بیشتر از یه ای پی بتونه وصل بشه بهش چیکار میشه کرد
چرا اضافه نمیکنید ؟

**Comment by hiddify-com** [comment, marked as answer]
به دلایل زیر قابل اجرا نیست:
- در پنل مالتی سرور، امکان تجمیع تعداد کانکشن ها نیست
- وقتی پشت کلود فلر میره عملا همه یه ای پی میشن و برای گرفتن آی پی اصلی پردازش بیشتری لازمه
- عملا تاثیر زیادی نداره جز اینکه کاربر اذیت میشه، محدودیت حجم به خودی خود کافی هست

**Reply by hatinati2** [reply]
سلام وقت بخیر
اول از تشکر می کنم از پنل عالی شما
واقعا از شما متشکرم
یه پیشنهاد داشتم برای اینکه برای تعداد کاربر برای یک کانفیک محدودیت بزاید
 به جای محدودیت ip  محدودیت اتصال با استفاده از uuid  بزارید 
 یعنی از هر uuid مثلا فقط یک بار بشه متصل شد

**Reply by hntcAir** [reply]
سلام وقت بخیر

محدودیت UUID نمیشه اعمال کرد سمت سرور؟


کانکشن همزمان را ساپورت نکنه
فقط تعداد کانکشن را در نظر بگیره، همین به نظر کافیه.


فرمودین محدودیت حجمی به خودی خود کافیه واقعا کافی نیست!

**Reply by hntcAir** [reply]
اگر پشت کلود نره و با RELAY هم باشه باز هم ایجاد این امکان غیرممکن است؟؟

**Comment by h-zare-dev** [comment]
سلام

دوستان با تجربه تر از من میگن کلادفلر گزینه ای برای real ip داره که اون رو ارسال می کنه به سرور با این راهکار قابل انجام نیست؟ 

پیشنهاد بعدی بنده استفاده از آدرس مک هست البته خیلی اطلاعات کاملی در این مورد ندارم

**Reply by SamSepiol-SS** [reply]
آدرس مک رو میشه جعل کرد

**Reply by mrgeex** [reply]
فکر نکنم یوزر معمولی جعل مک رو بدونه

**Comment by erfannjz** [comment]
میشه آیدی سخت افزاری دستگاه رو گرفت با اون کانفیگ رو ساخت مثه napsternetV
بعد به نرم افزار هامون اضافه کنیم
اگه با آیدی دستگاه مطابقت داشت کانفیگ کار کنه
عملا یه اکستنشن خاص برای فایلمون میخوایم که قفل باشه و با تکست ادیتور نشون نده
من تو گیتهابم یه مینی پروژه دارم که میاد آیدی سخت افزاری رو میگیره میتونید چکش کنید کمکی هم تو پایتون خواستین در خدمتم

**Reply by Farazzzssss** [reply]
سلام من دارم پایتون میخونم که یاد بگیرم میشه یه ایدی بدی تا اگه مشکلی داشتم ازت بپرسم

**Comment by conall1995** [comment]
برای مواردی که فرمودید قابل اجرا نیست 
میشه غیر از حالت مولتی سرور پیاده و فعال بشه 

برای مورد دوم که فرمودید پشت کلاد فلیر میشه از این خط کد توی nginx یا معادل توی apache و ... استفاده کرد که باعث میشه ip واقعی کاربر مشخص بشه ( proxy_set_header X-Real-IP $http_x_forwarded_for;)

برای مورد سوم هم خیلی وقتا نمیخوایم محدودیت حجمی بذاریم (متاسفانه محدودیت حجم توی ایران فقط خیلی بولد هست) و به نظر من شخصا تعداد نفراتی که استفاده میکنن اهمیت بیشتری داره

اگر بشه این قابلیت رو اضاف کنید کنار بقیه قابلیت های فوق العادتون عالی میشه ممنون

**Comment by aminamin1368** [comment]
واقعا این قابلیت خیلی مهمه اگه میشه اضافه کنید این قابلیت رو

**Comment by applesaeid** [comment]
خب چرا پنل های دیگری x-ui و یا ssrmu هر دو فابلیت کنترل کاربر دارن و پشت کلادفلر ایپی هست 
اگه بشه عالی میشه 
یعنی بهترین پنل هست پنلتون فقط این نبود کنترل کاربر خبلی اذیت میکنه اگه اونو داشت دیگه روش پنلی نبود
